handbook/engineering/security/index.md

```diff
@@ -2,6 +2,12 @@
 
 At Sourcegraph we think that security is an enabler for the business. Sourcegraph is committed to proactive security, and addressing vulnerabilities in a timely manner. We approach security with a can-do philosophy, and look to achieve product goals while maintaining a positive posture, and increasing our security stance over time.
 
+## Goals
+
+Customers know they can confidently and securely use Sourcegraph with their private code.
```

Comment:
(this change is good to merge as-is, but I wanted to clarify something)

This is more of a vision statement (which is valuable to have), not a goal (in the sense that I mean it).

When I say "goals" I mean more of what is currently in the internal "roadmap" document (which I would like to reframe as "goals" rather than "roadmap"). I see you took another pass at improving that document, which is great.

Here are examples of what "goals" look like for other teams:
- https://about.sourcegraph.com/handbook/product/goals
- https://about.sourcegraph.com/handbook/marketing#goals
- https://about.sourcegraph.com/handbook/engineering/code-intelligence#goals
- https://about.sourcegraph.com/handbook/engineering/distribution/goals

I'll leave some further comments on the doc itself.

Reply:
Agreed - we actually had that exact discussion, though I fell on the side of this being more goal than vision.  Let's continue this in that document.